Makefile

```diff
@@ -1,4 +1,5 @@
 GO ?= go
+export ARCH ?= amd64
```

Comment:
is there a reason why we can't use x86 directly?

Reply:
Usually x86 refers more to i386 , and amd64 the way to refer to x86_64
https://github.com/golang/go/blob/master/src/go/build/syslist.go
https://wiki.debian.org/SupportedArchitectures 
https://clang.llvm.org/docs/CrossCompilation.html

And in other places too

Reply:
However yes the libbpf parts we have expect __TARGET_ARCH_x86 and __TARGET_ARCH_i386